azure-monitor-opentelemetry/setup.py

```diff
@@ -86,15 +86,16 @@
     install_requires=[
         "azure-monitor-opentelemetry-exporter>=1.0.0b13",
         "opentelemetry-instrumentation~=0.38b0",
```

Comment:
Probably need to vendor `opentelemetry-instrumentation` as well.

Reply:
Done.